app.py

```diff
@@ -6,7 +6,15 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', items=items)
+    #return 'Hello World!'
+
+@app.route('/add', methods=['POST'])
+def addToDo():
+    title = request.form['title']
```

Comment:
This should probably fail if the title is empty (i.e. add some server side validation)